docs/en/latest/plugins/echo.md

```diff
@@ -45,7 +45,6 @@ This plugin addresses the corresponding functionality in the common phases such
 | body        | string | optional    |         |       | Body to replace upstream response.           |
 | after_body  | string | optional    |         |       | Body after the modification of filter phase. |
 | headers     | object | optional    |         |       | New headers for response                     |
-| auth_value  | string | optional    |         |       | Auth value                                   |
```

Comment:
We need to remove it from the source & test too.

Reply:
ok